browser/brave_wallet/solana_provider_browsertest.cc

```diff
@@ -305,6 +292,13 @@ bool WaitForWalletBubble(content::WebContents* web_contents) {
 
   return tab_helper->IsShowingBubble();
 }
+
+void CloseWalletBubble(content::WebContents* web_contents) {
```

Comment:
Had to add it to multiple Call* functions otherwise `WaitForWalletBubble` isn't working as what we expected because it was never dismissed after the previous request (we use NotifyXXXProcessed directly so UI doesn't really get dismissed).